terracumber_config/tf_files/tfvars/PR-TEST-uyuni.tfvars

```
@@ -0,0 +1,18 @@
+############ Uyuni unique variables ########################
+
+IMAGE                  = "opensuse154-ci-pro"
+IMAGES                 = ["rocky8o", "opensuse154o", "opensuse154-ci-pro", "ubuntu2204o"]
+SUSE_MINION_IMAGE      = "opensuse154o"
+SUSE_MINION_NAME       = "leap15"
```

```suggestion
SUSE_MINION_NAME       = "suse"
```
See https://gitlab.suse.de/galaxy/infrastructure/-/merge_requests/742